patches/TerrariaNetCore/Terraria/Terraria.csproj.patch

```diff
@@ -172,16 +124,59 @@
 -		<Message Text="Copying $(OutputName) to Steam Dir..." Importance="high" />
 -		<Copy SourceFiles="$(TargetPath)" DestinationFiles="$(TerrariaSteamPath)\$(OutputName).exe" />
 -		<Copy SourceFiles="$(TargetDir)$(OutputName).pdb" DestinationFolder="$(TerrariaSteamPath)" />
+-	</Target>
++
++	<!-- Import tasks -->
++	<UsingTask TaskName="SynchronizeDirectories" AssemblyFile="$(BuildTasksAssemblyFile)" TaskFactory="$(BuildTasksTaskFactory)" />
++	<UsingTask TaskName="OrganizeReferenceDestinations" AssemblyFile="$(BuildTasksAssemblyFile)" TaskFactory="$(BuildTasksTaskFactory)" />
++
++	<!-- Installs the build's output into the relevant game folder. -->
++	<Target Name="CopyToActualOutputDirectory" AfterTargets="Build">
++		<PropertyGroup>
++			<_ActualOutputDirectory>$(TerrariaSteamPath)</_ActualOutputDirectory>
++		</PropertyGroup>
++
++		<!-- Copy files systematically to output folder -->
++		<ItemGroup>
++			<BinFiles Include="$(TargetDir)**" />
++		</ItemGroup>
++
++		<Message Text="Copying $(AssemblyName) to '$(_ActualOutputDirectory)'..." Importance="high" />
++		<Copy SourceFiles="@(BinFiles)" DestinationFolder="$(_ActualOutputDirectory)/%(RecursiveDir)" SkipUnchangedFiles="True" />
++
++		<!-- Copy the Libraries folder, purging excessive files -->
++		<SynchronizeDirectories Source="$(TargetDir)Libraries" Destination="$(_ActualOutputDirectory)/Libraries" />
++	</Target>
++
++	<!-- Organize output copies of Project, NuGet, and directly referenced libraries under a single tidy folder. -->
++	<Target Name="OrganizeAllReferences" AfterTargets="ResolveAssemblyReferences" BeforeTargets="GenerateBuildDependencyFile;CopyFilesToOutputDirectory">
```

Comment:
Targets should be defined in the order they will be executed to reduce confusion for readers